issue in doc build while debian packaging

PYBUILD_BUILD_ARGS="PYTHONPATH={build_dir} http_proxy='localhost' sphinx-build -N -bhtml doc/source build/html" dh_auto_build
I: pybuild base:170: PYTHONPATH=/home/kieffer/workspace/pyFAI/package/pyFAI-0.12.1a0/.pybuild/pythonX.Y_2.7/build http_proxy='localhost' sphinx-build -N -bhtml doc/source build/html
Making output directory...
Running Sphinx v1.2.3

Exception occurred:
  File "conf.py", line 73, in <module>
IndexError: list index out of range
The full traceback has been saved in /tmp/sphinx-err-rp7d7z.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://bitbucket.org/birkenfeld/sphinx/issues/>. Thanks!
E: pybuild pybuild:256: build: plugin custom failed with: exit code=1: PYTHONPATH=/home/kieffer/workspace/pyFAI/package/pyFAI-0.12.1a0/.pybuild/pythonX.Y_2.7/build http_proxy='localhost' sphinx-build -N -bhtml doc/source build/html
dh_auto_build: pybuild --build -i python{version} -p 2.7 --dir . returned exit code 13
debian/rules:15: recipe for target 'override_dh_auto_build' failed
make[1]: *** [override_dh_auto_build] Error 13
make[1]: Leaving directory '/home/kieffer/workspace/pyFAI/package/pyFAI-0.12.1a0'
debian/rules:6: recipe for target 'build' failed
make: *** [build] Error 2
dpkg-buildpackage: error: debian/rules build gave error exit status 2
Build failed, please investigate ...

--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -65,12 +65,13 @@ copyright = u'2012-%s, Jerome Kieffer'% (year)
 
 # Configure the environment to be able to use sphinxcontrib.programoutput
 # NOTE: Must be done after pyFAI._version import which at the end of the end imports PyMCA
-# Importing MyMCA redefine and reorder PYTHONPATH
+# Importing PyMCA redefine and reorder PYTHONPATH
 import glob
 root_dir = os.path.abspath("../..")
 build_dir = glob.glob('../../build/lib*')
 os.environ["PATH"] = os.path.abspath(os.path.join(root_dir,"scripts")) + os.pathsep + os.environ.get("PATH","")
-os.environ["PYTHONPATH"] = os.path.abspath(build_dir[0]) + os.pathsep + os.environ.get("PYTHONPATH","")
+if build_dir:
+    os.environ["PYTHONPATH"] = os.path.abspath(build_dir[0]) + os.pathsep + os.environ.get("PYTHONPATH","")
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the

